test/system/groups_test.rb

```diff
@@ -42,7 +40,7 @@ class GroupsTest < ApplicationSystemTestCase
   end
 
   test 'destroying a Group' do
-    visit groups_url
+    visit project_group_path(@project, @group)
```

Comment:
It looks like there is a foreign key constraint preventing groups from being destroyed when they have unknowns associated to them.

```suggestion
    @group.update!(unknowns: create_list(:unknown, 2))
    visit project_group_path(@project, @group)
```